docs/wiki/Home.md

```diff
@@ -12,28 +12,17 @@ The following instructions assume that you have already forked **dcrdex**.
 
 <img src="images/wiki-creation.png" width="700">
 
-2. In your terminal, navigate to your local **dcrdex** directory and clone the wiki from the **decred/dcrdex** repo.
+2. In your terminal, navigate to your local **dcrdex** directory and add your remote wiki.
 
-`git clone https://github.com/decred/dcrdex.wiki.git wiki`
+```
+git remote add wiki https://github.com/your-username/dcrdex.wiki.git
+git subtree push --prefix docs/wiki wiki master
+```
 
-3. Create a remote for your fork's wiki, replacing 'your-username' with your username.
+3. Now after committing changes to the files in the docs/wiki folder, you can apply them to your dcrdex wiki and view immediately. Assuming you are in the root directory:
 
-`cd wiki; git remote add me https://github.com/your-username/dcrdex.wiki.git`
+```
+git push wiki `git subtree split --prefix docs/wiki HEAD`:master --force
```

Comment:
Cool learning about `subtree`!

Was looking at docs for the commands and came across tihs:

```
   push <repository> [+][<local-commit>:]<remote-ref>
       Does a split using the <prefix> subtree of <local-commit> and then
       does a git push to push the result to the <repository> and
       <remote-ref>. This can be used to push your subtree to different
       branches of the remote repository. Just as with split, if no
       <local-commit> is given, then HEAD is used. The optional leading + is
       ignored.
```

Sounds like `git subtree push` does a `split` and then a `git push`.  Doesn't that make this command the same as `git subtree push -f --prefix docs/wiki wiki HEAD:master` (where HEAD is actually implied if omitted)?

Reply:
Does -f force? Testing...

Reply:
HEAD was not necessary so removed.

> Except lets use $()

ok

Reply:
Using `$()`.